tests/end2end-tests/data_process/test.sh

```diff
@@ -353,6 +353,11 @@ rm -fr /tmp/np_remap/
 echo "********* Test the remap node emb/partial emb *********"
 python3 $GS_HOME/tests/end2end-tests/data_process/gen_emb_predict_remap_test.py --output /tmp/em_remap/
 
+# Add relation2id_map.json and rel_emb.pt into /tmp/em_remap/partial-emb/
+# remap_result should ignore this file
+echo 'Dummy' > /tmp/em_remap/partial-emb/relation2id_map.json
+echo 'Dummy' > /tmp/em_remap/partial-emb/rel_emb.pt
```

Comment:
If we just want files to be created we can use
```suggestion
touch /tmp/em_remap/partial-emb/relation2id_map.json
touch /tmp/em_remap/partial-emb/rel_emb.pt
```